src/one_dragon/base/push/channel/work_weixin.py

```
@@ -0,0 +1,201 @@
+            # 判断整体结果
+            if text_success and image_success:
+                return True, "推送成功"
+            elif text_success or image_success:
+                return True, f"部分推送成功: {'; '.join(error_messages)}" if error_messages else "部分推送成功"
+            else:
+                return False, f"推送失败: {'; '.join(error_messages)}" if error_messages else "推送失败"
+
+        except Exception as e:
+            return False, f"企业微信推送异常: {str(e)}"
+
+    def _send_image(self, url: str, headers: dict[str, str], image: MatLike, error_messages: list[str]) -> bool:
```

Ruff (0.14.2)

Undefined name `MatLike`

(F821)